probably wrong ?

hvysakhan opened the pull request:
added a strto dec convertor in one pass

hvysakhan committed:
probably wrong ?

ghost commented:
Again, make sure you read [this link](https://github.com/Islandora/islandora/wiki/Git-Guidelines-and-Best-Practices) on how to make meaningful commits. Or just run `git log` to see previous commit messages to get a general idea.

For now, for me to merge it, I need you to

Change the commit message by `git commit --amend`
Then `git push`. It will automatically come as a new commit on this page. :+1:

ghost commented:
Btw commit message is vague, I didn't understand... :disappointed: 
Also it's Blesswin's code :lollipop: 
Try `git blame [filename]` on the original absloader file :wink:

hvysakhan committed:
stringtodec convertor added
to one pass assembler
added a program to find palindrome of a number using recursion

hvysakhan committed:
made  changes to one pass and also introduced new program
stringtodec convertor added to one pass assembler
added a program to find palindrome of a number using recursion

hvysakhan committed:
Merge branch 'master' of https://github.com/hvysakhan/codesalad

ghost commented:
I'm back ...
